pkgs/development/python-modules/sslyze/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, lib, callPackage, fetchPypi, fetchFromGitHub, buildPythonPackage, nassl, cryptography, cryptography_vectors, typing-extensions, faker }:
+buildPythonPackage rec {
+  name = "sslyze";
+  version = "3.0.4";
+  src = fetchFromGitHub {
+    owner = "nabla-c0d3";
+    repo = name;
+    rev = version;
+    sha256 = "19fnnbdmpfdpmadqqcnj79hh72wrjiahn58rbxis4586ydyi552g";
+  };
+
+  patchPhase = ''
+    sed -i 's/cryptography>=2.6,<=2.9/cryptography>=2.6,<=${cryptography.version}/g' setup.py 
+  '';
+
+  propagatedBuildInputs = [ nassl cryptography typing-extensions faker ];
+
+  meta = with lib; {
+    homepage = "https://github.com/nabla-c0d3/sslyze";
+    description = "Fast and powerful SSL/TLS scanning library";
+    platforms = platforms.linux ++ platforms.darwin;
+    license = licenses.agpl3;
+    maintainers = maintainers.veehaitch;
```

usually a list (although i think the single item still works, technically)
```suggestion
    maintainers = maintainers; [ veehaitch ];
```